index.js

```diff
@@ -113,6 +110,6 @@ function close (fastify, done) {
 }
 
 module.exports = fp(fastifyWebsocket, {
-  fastify: '3.x',
+  fastify: '>= 3.10.1',
```

Comment:
This does not qualify as a breaking change?

Reply:
Nope, in a real world application you should use lock files, so this wouldn't impact you until you want to upgrade.